### PR TITLE
Docs: Add blog post for 1.3.0 announcement

### PR DESCRIPTION
**Pull request by rxaviers**

### CODE REVIEW

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+// > [
+//   { "type": "month", "value": "11" },
+//   { "type": "literal", "value": "/" },
+//   { "type": "day", "value": "30" },
+//   { "type": "literal", "value": "/" },
+//   { "type": "year", "value": "2010" }
+// ]
+```
+
+The information is made available separately and it can be formatted and concatenated again in a customized way. For example by using [`Array.prototype.map()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map), [arrow functions](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions), a [switch statement](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/switch), [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals), and [`Array.prototype.reduce()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce).
```
Comment (jzaefferer) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Looks like its not using reduce anymore, and the use of the switch statement is pretty odd - an if would do just fine.

I don't think `.map` needs an MDN link.

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+| Localized and smart date input           | Feb 28 in `en`, `es`, `pt`, `de`, `zh`, `ko`, and `ar` |
+| ---------------------------------------- | ---------------------------------------- |
+| ![en](https://media.giphy.com/media/xUA7aZAUNINGP2jI4M/giphy.gif) | ![en-es-pt-de-zh-ko-ar](https://media.giphy.com/media/3og0ILQu0KxLRewJnW/giphy.gif) |
+
+## Dynamically Augment Date Skeletons
+
+> This change was contributed by Marat Dyatko @vectart and Artur Eshenbrener @Strate in #462 and #604.
+
+The style used to display a date format often vary depending on the application. CLDR offers data for certain presets like short, medium, long, and full vs. date, time, and datetime. It also offers data for a lot of different [date fields](http://www.unicode.org/reports/tr35/tr35-dates.html#Date_Field_Symbol_Table) and their combinations, which are used by globalize to sintetize an open-ended list of custom formats (called skeletons). But, what's interesting is that it would be prohibitively large if CLDR provided data for every single possible combination. So, there's an algorithm specified by UTS#35 to deduce missing data from the requested format.
```
Comment (jzaefferer) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: The first sentence has some issue, but I'm not sure how to fix it. Maybe "style" (singular) and "vary" (expecting plural) don't go together?

Comment (jzaefferer) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Typo in "sintetize", I guess it should be "synthesise"?

Comment (jzaefferer) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: For anyone not well familiar with the feature, a quick example of skeletons would help.

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+For example, there are four different presets to format a date and they would generate `"7/1/17"` (short), `"Jul 1, 2017"` (medium), `"July 1, 2017"` (long), and `"Saturday, July 1, 2017"` (full). Although, let's pretend we want something different such as `"Jul 1"`. For that format we could use `{skeleton: "MMMd"}`. Internally, globalize finds a direct match (in CLDR) for that skeleton. Let's now pretend we want `"July 1"`. For that format we could use `{skeleton: "MMMMd"}`. Internally, globalize doesn't find a direct match (in CLDR) for this skeleton and it needs to use the data for `MMMd`, which in the case of English maps to `"MMM d"`, and then replace `MMM` with `MMMM` dynamically generating `"MMMM d"`. If we wanted `"07/01"` instead, we could use `{skeleton: "MMdd"}`. Internally, globalize doesn't find a direct match in CLDR for this skeleton and it needs to use the data for `Md`, which in the case of English maps to `"M/d"`, and then replace `M` wtih `MM` and `d` with `dd` dynamically generating `"MM/dd"`.
```
Comment (jzaefferer) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: I asked for examples above, which this does indeed deliver, but it seems to mix old and new features together. Its not clear to me which part is new, and which already existed before. That's fine for docs, but for a release blog post, it should clarify the difference.

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+To make a long story short, the algorithm in globalize 1.3.0 has been significantly improved and it allows using virtually any skeletons except for some special cases.
+
+```js
+let skeleton = "GyMMMMEEEEdhms";
```
Comment (jzaefferer) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Related to previous comments, what would this output before 1.3.0?

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+// > 'السبت، ١ يوليو، ٢٠١٧ م ٥:٠١:٤٠ م'
+Globalize("ar-MA").formatDate(new Date(), {skeleton});
+// > 'السبت، 1 يوليوز، 2017 م 5:04:29 م'
+Globalize("it").formatDate(new Date(), {skeleton});
+// > 'sabato 1 luglio 2017 d.C. 5:01:52 PM'
+```
+
+Read our [getting started](https://github.com/globalizejs/globalize/#getting-started) and play with it yourself.
+
+## Other Enhancements and Bug Fixes
```
Comment (jzaefferer) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Not sure in what format this is supposed to be published, but currently all the PR and issue references aren't links. Above there's enough content to ignore that, but the following is lacking context to be useful without links.

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md (hunk 2 above)
Comment (dwbruhn) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Yeah, "vary" --> "varies"

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md (hunk 5 above)
Comment (rxaviers) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Add globalize 1.2.3 context wrt skeleton - 8919fdfa

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md (hunk 4 above)
Comment (rxaviers) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Add globalize 1.2.3 context wrt skeleton - 8919fdfa

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md (hunk 3 above)
Comment (rxaviers) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Add globalize 1.2.3 context wrt skeleton - 8919fdfa

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md (hunk 2 above)
Comment (rxaviers) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Varies and synthesise update  - 8919fdfa

Comment (rxaviers) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: synthesise -> synthesize - 97f02bac

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md (hunk 5 above)
Comment (rxaviers) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Yeap good point! Update gh links - 8bb71bdd

**Comment by rxaviers** [comment]
Thank you @jzaefferer! Very helpful!

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md [hunk 6]
```diff
@@ -0,0 +1,167 @@
...
+// > 'domingo, 19 de março de 2017 19:19:22 Horário Padrão de Brasília'
+Globalize("de").formatDate(date, {datetime: "full", timeZone: "Europe/Berlin"});
+// > 'Sonntag, 19. März 2017 um 23:19:22 Mitteleuropäische Normalzeit'
+Globalize("zh").formatDate(date, {datetime: "full", timeZone: "Asia/Shanghai"});
+// > '2017年3月20日星期一 中国标准时间 上午6:19:22'
+Globalize("ar").formatDate(date, {datetime: "full", timeZone: "Africa/Cairo"});
+// > 'الاثنين، ٢٠ مارس، ٢٠١٧ ١٢:١٩:٢٢ ص توقيت شرق أوروبا الرسمي'
+```
+
+We believe to have solved this in a lighweight manner by using [zoned-date-time](https://github.com/rxaviers/zoned-date-time) under the hoods, which is a 0.6KB library for the time zone manipulations, and by leveraging the Globalize Compiler for precompling the IANA data base for production. For example, let's say you are serving content in English for *America/Los_Angeles* time using the following formatter:
```
Comment (patch) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Typos:
* lighweight → lightweight
* precompling → precompiling

**Comment by rxaviers** [comment]
Thanks - fixed typos in c87b19ea

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md (hunk 6 above)
Comment (rxaviers) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: Thanks - fixed typos in c87b19ea

File: doc/blog-post/2017-07-xx-1.3.0-announcement.md [hunk 7]
```diff
@@ -4,7 +4,7 @@ Today we are releasing Globalize 1.3.0. It is a special release, because it incl
 
 ## IANA/Olson Time Zone Support
 
-> This change was contributed by Ramalingam Manikandan @rajavelmani (PayPal) and Rafael Xavier @rxaviers in #687 and #701.
+> This change was contributed by Kandaswamy Manikandan @rajavelmani (PayPal) and Rafael Xavier @rxaviers in #687 and #701.
```
Comment (rajavelmani) [inline doc/blog-post/2017-07-xx-1.3.0-announcement.md]: 👍

**Pull request merged**